modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -6,7 +6,11 @@
 [id="rosa-required-aws-service-quotas_{context}"]
 = Required AWS service quotas
 
-The table below describes the AWS service quotas and levels required to create and run an {product-title} cluster.
+The table below describes the AWS service quotas and levels required to create and run one {product-title} cluster. While most default values are suitable for most workloads, you are likely to need to request additional quota for the following cases:
+
+* ROSA clusters require at least 100 vCPUs, but the default maximum value for vCPUs assigned to Running On-Demand Standard Amazon EC2 instances is `5`. Therefore if you have not created a ROSA cluster using the same AWS account previously, you must request additional EC2 quota for `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances`.
+
+* Some optional cluster configuration features, such as custom security groups, may require that you request additional quota. For example, because ROSA uses 3 security groups by default, and the default quota for `Security groups per network interface` is `5`, up to 2 custom security groups can be added to a cluster without requesting additional quota for `Security groups per network interface`.
```

Comment:
This isn't quite right.

We create 3 security groups and attach them to the VPC but not the network interfaces. A network interface is created and associated with each new worker machine pool, but we only attach the worker-security-group to each worker machine pool and the user would have 4 extra security groups to use without any changes to AWS quota.

Reply:
Thanks @gdbranco! Based on the new maximum of 5 additional security groups this would mean that users can add up to 4 additional SGs without changing quota, and only need to increase quota if they want to add a 5th additional SG, correct?

Reply:
I've reworded this to the following:

* Some optional cluster configuration features, such as custom security groups, may require that you request additional quota. For example, because ROSA associates the `worker-security-group` security group with network interfaces in worker machine pools by default, and the default quota for `Security groups per network interface` is `5`, up to 4 additional custom security groups for worker nodes can be added to the cluster without requesting additional quota for `Security groups per network interface`.

Reply:
CC @bchandra-ocp @gdbranco Please confirm whether we are supporting up to 5 security groups total or up to 5 additional security groups

Reply:
Confirmed, 5 additional.